scripts/config.ts

```diff
@@ -264,11 +282,20 @@ function writeConfigs(argv: any) {
         simpleConfig.node["batch-poster"].enable = true
         simpleConfig.node["batch-poster"]["redis-url"] = ""
         simpleConfig.execution["sequencer"].enable = true
+        if (argv.anytrust) {
+            simpleConfig.node["data-availability"].enable = true
```

Comment:
Thanks, that makes it simpler. I don't need to make `rest-aggregator.enable` conditional since it can just be marked enabled, and be controlled by whether or not `data-availability.enable` is set.